website/docs/feature-flag-tutorials/use-cases/ai.md

```diff
@@ -3,7 +3,9 @@ title: How to use feature flags with AI
 slug: /feature-flag-tutorials/use-cases/ai
 ---
 
-Many tech companies today are integrating generative AI with large language models (LLMs) into their products, often in the form of chatbots or content generators.
+Hello,
```

Comment:
I think I preferred it without the Hello.

I think this intro bit can live without a heading (as it is now), but I would probably add the first heading before we start doing any actual work. Like above the section that starts with 'We'll start with a basic chatbot... something like "Initial setup"? maybe?